### PR TITLE
Query-update setting

### PR DESCRIPTION
**Pull request by Bleu-Yves-Sopoude**
In this feature we:

- Added a column species of type string to the animals' table
- Inserted data in the table
- Updated data with transactions
- Wrote queries to answer the given questions.

ts1
ts3
ts4
ts5
ts6

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -6,3 +6,16 @@ VALUES('Agumon', '2020-02-03', 0, true,10.23),
      ('Gabumon', '2018-11-15', 2, true,8),
      ('Pikachu', '2021-01-07', 1, false,15.04),
      ('Devimon', '2017-05-12', 5, true,11);
+
+
+-- Query and Update animals table
+
+
+INSERT INTO animals(name,date_of_birth, escape_attempts,neutered,weight_kg,species) 
+VALUES('Charmander', '2020-02-08', 0, true,-11,NULL),
```
Comment (BregornOriginal) [inline data.sql]: - [ ] Kindly fix the value of `escape_attemps` for Charmander, it's false

File: data.sql [hunk 2]
```diff
@@ -6,3 +6,16 @@ VALUES('Agumon', '2020-02-03', 0, true,10.23),
...
+     ('Plantmon', '2021-11-15', 2, true,-5.7,NULL),
+     ('Squirtle', '1993-04-02', 3, false,-12.13,NULL),
+     ('Agemon', '2005-06-12', 1, true,-45,NULL),
```
Comment (BregornOriginal) [inline data.sql]: - [ ] Kindly fix the name of this animal, it's `Angemon`

File: queries.sql [hunk 3]
```diff
@@ -16,3 +16,66 @@ SELECT * FROM animals WHERE neutered=true;
...
+
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT SP2;
+
+UPDATE animals SET weight_kg=weight_kg * (-1);
+
+ROLLBACK TO SP1;
+
+UPDATE animals SET weight_kg=weight_kg * (-1) WHERE weight_kg < -1;
```
Comment (BregornOriginal) [inline queries.sql]: - [ ] Excellent work implementing queries! Regarding this one, kindly remember to multiply only the weight_kg where animals are `negative` it's include -1 🥇 Kindly fix this one

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @Bleu-Yves-Sopoude  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

giphy (4)

## Highlights

✅ Linters are passing
✅ README looks professional 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @Bleu-Yves-Sopoude , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some that haven't been addressed yet.


 ## Highlights :fire: 
- Most requested changes implemented :heavy_check_mark: 


 ## Required Changes ♻️
- [ ] Great job implementing the previous reviewer's suggestions :clap: However, your screenshot queries still show previous results and not the current results with the made changes so I am unable to confirm if they have been implemented in your database. So kindly add current screenshots of your queries to your pull request after making the requested changes :wink:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Bleu-Yves-Sopoude 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- You have updated the screenshots with correct data :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**